blog/_posts/2016-10-16-B-More-Stunners-Over-and-Out.md

```diff
@@ -0,0 +1,69 @@
+---
+title: B'More Stunners&#58; B'More Stunners&#58 Over and Out!
+layout: post
+created_at: Sunday October 16, 2016
+permalink: blog/2016-10-16-B-More-Stunners-Over-and-Out
+current: blog
+author: Oreoluwa Akinsanmi and Ashley Jean
+twitter: ashhjean 
+categories:
+- student-posts
+---
+
+
+![teamphoto](/img/blog/2016/ashley.jpeg)
```

Comment:
All images need image credits, as clearly explained [here](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images). It's enough to put down your team name if you took the picture — the name of the person who took the photo is ok too.